commands/host.vegman

```diff
@@ -236,3 +236,21 @@ If connected via SSH, add extra ~ characters, one per SSH connection.
 EOF
   exec obmc-console-client
 }
+
+# @sudo cmd_config admin
+# @restrict cmd_config admin
+# @doc cmd_config
+ # Host configuration
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_pcie admin
+# @restrict cmd_config_pcie admin
+# @doc cmd_config_pcie
+# Configure PCIe bifurcation
+#  Use -h or --help to show usage
```

Comment:
We don't need this, all arguments (include -h) will be passed to host-pcie-cfg.

Reply:
ok